README.md

```diff
@@ -7,7 +7,7 @@ Below is a sample command line that shows how the field mosaic Docker image coul
 An explanation of the command line options used follows.
 Be sure to read up on the [docker run](https://docs.docker.com/engine/reference/run/) command line for more information.
 
-```docker run --rm --mount "src=/home/test,target=/mnt,type=bind" agpipeline/ply2las:2.1 --working_space "/mnt" --metadata "/mnt/08f445ef-b8f9-421a-acf1-8b8c206c1bb8_metadata_cleaned.json" ```
+```docker run --rm --mount "src=/home/test,target=/mnt,type=bind" agpipeline/ply2las:2.1 --working_space "/mnt" --metadata "/mnt/08f445ef-b8f9-421a-acf1-8b8c206c1bb8_metadata_cleaned.json" /mnt/3c807fe1-a5ba-4b4b-b618-1d2c9c981678__Top-heading-east_0.ply```
```

Comment:
```suggestion
```sh
docker run --rm --mount "src=/home/test,target=/mnt,type=bind" agpipeline/ply2las:2.1 --working_space "/mnt" --metadata "/mnt/08f445ef-b8f9-421a-acf1-8b8c206c1bb8_metadata_cleaned.json" /mnt/3c807fe1-a5ba-4b4b-b618-1d2c9c981678__Top-heading-east_0.ply
```
```

Reply:
Why give an example but not provide example files (e.g. test files)?

Reply:
Good point. Will add